README.md

```diff
@@ -23,14 +23,34 @@ Include the SAML library.
 To use the saml library, we think in terms of service providers (e.g. Clever) and identity providers (e.g. partners that use ADFS).
```

Comment:
Also the first section (the description of the library) seems outdated/incorrect. 

> Takes care of the complexities of the SAML protocol and provides an easy interface for using it. Specifically, creating metadata.xml files, creating AuthnRequests and parsing and validating AuthnResponses.

> This is exposed as both a series of functions that implement each step of the SAML protocol, and an Express middleware that creates the necessary endpoints for the metadata, the login and the assertion.

Lets change it to be more correct